test/e2e/framework/sessionconfig.go

```diff
@@ -80,6 +81,22 @@ func (cfg SessionConfig) outerHeaderRemoval() *ie.IE {
 	return ie.NewOuterHeaderRemoval(pfcp.OuterHeaderRemoval_GTPUUDPIPV6, 0)
 }
 
+const (
+	BBF_EID                 = 3561
```

Comment:
It would be nice to have comments with a bit of explanation here, such as pointer to the section in the spec

Reply:
Added an explanation